packages/components/src/pricing-slider/types.ts

```diff
@@ -37,24 +43,26 @@ export type PricingSliderProps = {
 
 	/**
 	 * Callback called on every value change.
-	 * The function will be called with two arguments, the first being the new value(s) the second being thumb index.
 	 */
 	onChange?: ( value: number ) => void;
 
 	/**
-	 * Callback called before starting to move a thumb. The callback will only be called if the action will result in a change.
-	 * The function will be called with two arguments, the first being the initial value(s) the second being thumb index.
+	 * Callback called before starting to move a thumb.
 	 */
 	onBeforeChange?: ( value: number ) => void;
 
 	/**
-	 * Callback called only after moving a thumb has ended. The callback will only be called if the action resulted in a change.
-	 * The function will be called with two arguments, the first being the result value(s) the second being thumb index.
+	 * Callback called only after moving a thumb has ended.
 	 */
 	onAfterChange?: ( value: number ) => void;
 
 	/**
 	 * Node to render on the slider.
 	 */
 	renderThumb?: RenderThumbFunction;
+
+	/**
+	 * The marks on the slider, represented as an array of numbers.
+	 */
+	marks?: number[];
```

Comment:
Let's update this to `marks?: boolean | number[];`. We might even consider just `boolean` for now if we only want to expose the features we are actually using.